README.md

```diff
@@ -7,7 +7,7 @@ A simple collection of custom Hamcrest matchers for use with `hamcrest/hamcrest-
 
 Install with composer:
 
-`composer require garethellis/hamcrest-matchers`
+`composer require --dev garethellis/hamcrest-matchers`
```

Comment:
Probably worth noting that since Hamcrest *can* be used in application code, you can install it as a non-dev dependency but that for most use cases `--dev` will suffice.